onadata/apps/logger/views.py

```diff
@@ -504,8 +503,8 @@ def delete_xform(request, username, id_string):
         {"user__username__iexact": username, "id_string__iexact": id_string}
     )
 
-    # delete xform and submissions
-    remove_xform(xform)
```

Comment:
Should we also delete the function `remove_xform` since it is no longer required?

Reply:
done!